index.html

```diff
@@ -0,0 +1,506 @@
+          <a href="#" class="text-primary_blue font-semibold hover:text-black cursor-pointer">Pricing details</a>
+        </div>
+        <!-- </div> -->
+      </div>
+    </section>
+
+
+
+
+
+
+
+    <!--footer-->
+
+    <section class="mt-10 relative">
+      <div class="-skew-y-6  w-full bg-slate-100  h-[13vw] absolute z-10 top-0">
+        <div class="flex justify-between -mt-28  relative z-40">
+          <!-- item 1 -->
+          <div class="hidden lg:block -mt-24">
+            <div class="relative bg-cyan-400 -skew-y-6 w-52 h-10  ml-10  mt-40"></div>
+            <div class="bg-primary_blue   opacity-80 -skew-y-6 w-80 h-12  -mt-6 "></div>
+          </div>
+          <!-- item 2 -->
+          <div class="hidden lg:block mt-32">
+            <div class="relative bg-cyan-400 -mt-16 -skew-y-6 w-80 h-10    "></div>
+            <div class="bg-primary_blue relative  opacity-80 -skew-y-6 w-52 h-10 ml-10  -mt-4"></div>
+          </div>
+        </div>
+      </div>
+      <div class="bg-slate-100 relative top-[7vw] bg-no-repeat bg-center bg-cover md:px-28px">
+        <div class="md:px-32 px-8 py-20">
+          <div class="mt-8 grid grid-cols-2 gap-12 md:grid-cols-4 md:gap-4">
+
+            <!--1-->
+            <div class="flex flex-col gap-2 col-span-2 md:col-span-1 md:justify-between">
+              <div>
+                <img class="w-16 cursor-pointer " src="./assets/images/logo_white.svg" alt="">
+                <div class="flex items-center gap-2 md:mt-2">
+                  <svg class="LocaleControl__triggerIcon" width="16" height="16" viewBox="0 0 16 16">
+                    <path
+                      d="M3.54 8.04h4.42v4.43c0 .57.76.75 1.01.24l4.96-9.93a.54.54 0 0 0-.72-.72L3.3 7.03c-.5.25-.33 1.01.24 1.01"
+                      fill="#0A2540" fill-rule="evenodd"></path>
+                  </svg>
+                  <h4>United states</h4>
+                </div>
+                <div class="flex items-center gap-2">
+                  <svg class="LocaleControl__triggerIcon" width="16" height="16" viewBox="0 0 16 16">
+                    <path
+                      d="M7.5 2C10.91 2 13 4.01 13 6.5c0 2.07-1.45 3.82-3.92 4.34L6.62 13.3a.66.66 0 0 1-1.12-.46v-2.1C3.29 10.12 2 8.45 2 6.5 2 4.01 4.09 2 7.5 2z"
+                      fill="#0A2540" fill-rule="evenodd"></path>
+                  </svg>
+                  <h4>English (United States)</h4>
+                </div>
+              </div>
+
+              <h4>© 2022 Stripe, Inc.</h4>
+            </div>
+
+            <!--2-->
+            <div class="flex flex-col gap-8">
+              <div class="flex flex-col gap-2">
+                <h4 class="font-semibold text-gray-700 text-sm">Products</h4>
+                <a class="hover:text-dark_gray" href="#">Atlas</a>
+                <a class="hover:text-dark_gray text-sm" href="#">Billing</a>
+                <a class="hover:text-dark_gray text-sm" href="#">Capital</a>
```

Comment:
Fix this stripe footer logo, check your page and the original page: 
Yours: 
![image](https://user-images.githubusercontent.com/58233753/203081700-126be550-dd4d-474f-9b5b-4e34a32fff94.png)
Original
![image](https://user-images.githubusercontent.com/58233753/203081513-26572580-503c-4c30-bd7d-e8355e7c316c.png)